packages/flutter_tools/lib/src/test/flutter_web_platform.dart

```diff
@@ -735,6 +735,7 @@ class BrowserManager {
 
         _controllers.add(controller);
         return await controller.suite;
+      // Not limiting to catching Exception because the exception is rethrown.
```

Comment:
`finally` over `catch`-`rethrow`?

Reply:
The `finally` clause would run even if no exception is thrown, which would change the semantics.

Reply:
Ahh, I see.